Aws page (#592)
gravitational/next: pages/use-cases/aws/index.mdx

      </>
    }
>
    <ProductBanner.Item iconWrapper title="Infrastructure sprawl" src="./assets/security-icon.svg">
      You need to control who can provision infrastructure to control costs and prevent sprawl.
    </ProductBanner.Item>
    <ProductBanner.Item iconWrapper title="Granular access" src="./assets/complexity-icon.svg">
      Different teams require different levels of identity-based access for different resources.
    </ProductBanner.Item>
    <ProductBanner.Item iconWrapper title="Security &amp; Compliance" src="./assets/security-icon.svg">
      You need to lock down who can provision & access your AWS infrastructure and keep detailed audit logs. 
    </ProductBanner.Item>
  </ProductBanner>
</Section>

<CenterCard 
  src="./assets/security-icon.svg" 
  title="Why AWS customers big and small trust Teleport" 

>   text="Our unique approach to AWS infrastructure access is not only more secure &mdash; it actually improves developer productivity.  By providing an identity-aware AWS infrastructure access solution that developers love to use, you can easily implement security and compliance without worrying about backdoors that outmoded solutions encourage."/>

Reviewer: This feels very verbose..

<Section>
  <Feedback reviews={meta.reviews} />
</Section>

<Section width="100%" bg="flatWhite">
  <ProductBanner

>     subtitle="Get the most our of AWS Console"

Reviewer: ```suggestion
    subtitle="Get the most out of AWS Management Console"
```

    title="Control who can provision AWS resources"
    width="100%"
    src="./assets/security-that-scales-with-you@2x.png"
    imgPosition="right"
    logoSrc={startups}
    description={
      <>
        With Teleport, you can separate who can provision AWS resources from who can access them.
        We provide an identity-aware access solution for AWS Console and the AWS API that enables fine-grained, role-based access controls to prevent unauthorized creation of AWS resources.  If an SRE needs to be able to provision new AWS resources, you can easily grant temporary elevated access that automatically expires.
      </>
    }
  >

>     <ProductBanner.Item title="AWS Console">

Reviewer: ```suggestion
    <ProductBanner.Item title="AWS Management Console">
```

    logoSrc={secteams}
    description={
      <>
       The Teleport Cloud runs on AWS, so whether you use our hosted control plane, or self-host on AWS, you get the same integrated experience.
      </>
    }
  >
    <ProductBanner.Item title="Hosted on AWS">
      The Teleport Cloud is hosted across multiple AWS regions so you can run your AWS workloads close to your Access Plane.
    </ProductBanner.Item>
    <ProductBanner.Item title="Buy on AWS Marketplace">
      Purchase Teleport directly via the AWS Marketplace or use AWS credits to pay for Teleport usage.  Private offers available for enterprise customers.
    </ProductBanner.Item>
    <ProductBanner.Item title="AWS CloudHSM">
    For added security protecting the Teleport Certificate Authority (CA) running on AWS, we support AWS CloudHSM.
    </ProductBanner.Item>
  </ProductBanner>
</Section>

<Section bg="flatWhite">
  <WorksWith title="Works with everything you have">
    Teleport Server Access is open source and it relies on open standards such
    as X.509 certificates, HTTPS, SAML, OpenID connect and others. Deployed as a
    single-binary, it seamlessly integrates with the rest of your stack.
  </WorksWith>

> </Section>

Reviewer: Can we modify these icons to add CloudFormation and Terraform? ( instead of Chef & Puppet )

<EasyStart>
      <Terminal>
```
# on a client
$ tsh login --proxy=example.com

> # on a server

Reviewer: ```suggestion
# on AWS Linux 2
```

> $ apt install teleport

Reviewer: ```suggestion
$ sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
$ sudo yum install teleport
```

This command is more RHEL specific.
